projects/packages/publicize/src/class-publicize-utils.php

```diff
@@ -109,4 +109,14 @@ public static function assert_is_wpcom( $method ) {
 			throw new \Exception( esc_html( "Method $method can only be called on WordPress.com." ) );
 		}
 	}
+
+	/**
+	 * Check if the new module endpoint is available in the used Jetpack version.
+	 *
+	 * @return bool
+	 */
+	public static function has_new_module_endpoint() {
+		// @phan-suppress-next-line PhanTypeMismatchArgumentNullableInternal - Phan thinks JETPACK__VERSION is not a string (it is).
+		return class_exists( 'Jetpack' ) && defined( 'JETPACK__VERSION' ) && ( version_compare( JETPACK__VERSION, '14.3', '>=' ) );
```

Comment:
We could cast it to make Phan happy? And it ensures that it stays a string

```suggestion
		return class_exists( 'Jetpack' ) && defined( 'JETPACK__VERSION' ) && ( version_compare( (string) JETPACK__VERSION, '14.3', '>=' ) );
```

Reply:
I tried but I also got the error like that. I can cast it too to be sure

Reply:
Added in 0df6110

Reply:
Is it that it thinks that JETPACK__VERSION could be null?